src/Analyzers/Analyzers.csproj

```diff
@@ -2,30 +2,22 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeShared>false</IncludeShared>
     <IsRoslynComponent>true</IsRoslynComponent>
     <EnableStyleCop>true</EnableStyleCop>
-
-    <!-- Do not include the generator as a lib dependency -->
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-    <IncludeShared>false</IncludeShared>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <!-- Do not publish it until it is ready to release -->
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <PackageDescription>.NET Analyzers for the Durable Task Framework.</PackageDescription>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);SA0001;</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>
     <VersionPrefix>0.0.1</VersionPrefix>
     <VersionSuffix>preview.1</VersionSuffix>
+    <PackageDescription>.NET Analyzers for the Durable Task Framework.</PackageDescription>
     <NeutralLanguage>en</NeutralLanguage>
+    <IncludeBuildOutput>false</IncludeBuildOutput> <!-- Do not include the analyzer as a lib dependency -->
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <IsPackable>false</IsPackable> <!-- Do not publish it until it is ready to release -->
```

Comment:
I see that you're just reordering this, but why do we set this to `false`? I assume we only generate the package anyways when we do `dotnet pack` and I'm wondering why we'd want to block someone from using `dotnet pack`. Let me know if I'm misunderstanding.

Reply:
Our CI pipeline runs `dotnet pack` on the solution file. To avoid this from being included in that step and being unintentionally released, we disabled packing. If someone wants to pack locally, they can override this behavior with `-p:IsPackable=true`